geode-core/src/main/java/org/apache/geode/internal/net/SocketCreator.java

```diff
@@ -204,10 +200,10 @@
             String addrName = reverseDNS(addr);
             if (inetAddress.isLoopbackAddress()) {
               inetAddress = addr;
-              lhName = addrName;
+              break;
```

Comment:
the old condition for terminating the loop was if lhName != null. lhName is the result of calling reverseDNS which can return null. So in the new code shouldn't this "break;" be in "if (addrName != null)"?

Reply:
Good find. I will fix.